dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -322,14 +349,93 @@ object Engine {
       }
       Some(sc)
     } else {
-      physicalCoreNumber = cores
+      _onSpark = false
```

Comment:
We can add an option in bigdl.sh to set that env var. So users can change it easily